test_requirements.txt

```diff
@@ -4,53 +4,53 @@
 #
 #    pip-compile --output-file=test_requirements.txt test_requirements.in
```

Comment:
You now have `--upgrade` in `requirements.txt` but not here.. is this intentional?

Reply:
Not really, besides that we had that before :-p But yeah, il put it in both. But I think pip-tools removes it in the printout again, so it is an eternal battle keeping it in there :-p